torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -1068,6 +1067,21 @@ def init_variable(
                         "use_cuda_graph" in defaults
                         and defaults["use_cuda_graph"].default != kernel.use_cuda_graph
                     )
+                    # pre_hook requires running arbitrary code at runtime, which we cannot handle at this time
+                    # https://github.com/pytorch/pytorch/issues/139059
+                    or (
+                        # Check Config passed to autotuner in configs
+                        any(cfg.pre_hook is not None for cfg in kernel.configs)
```

Comment:
Should we check this separately to make the error message more specific?

Reply:
yeah the only configs & keys one is a bit vague. maybe something like "pre_hook is not supported in triton.Autotune Configs"?